fe/fe-core/src/main/java/org/apache/doris/load/ExportJob.java

```diff
@@ -109,7 +109,7 @@ public class ExportJob implements Writable {
 
     private static final int MAXIMUM_TABLETS_OF_OUTFILE_IN_EXPORT = Config.maximum_tablets_of_outfile_in_export;
 
-    public static final String CONSISTENT_ALL = "all";
```

Comment:
Does this have upgrade or down grade risk??
For example , in old version user create a job and using CONSISTENT_ALL, then the property is persistent. And then upgrade to new version, will not find the related property key.